docs/_data/schema-config.yaml

```diff
@@ -143,11 +144,15 @@ about-details:
   projector_ratio: str(required=False)
 
 cfp-details:
-  url: str(required=False)
-  ends: str(required=False)
-  notification: str(required=False)
+  url: str(required=True)
+  ends: str(required=True)
+  notification: str(required=True)
 
 grants-details:
-  url: str(required=False)
-  ends: str(required=False)
-  notification: str(required=False)
+  url: str(required=True)
+  ends: str(required=True)
+  notification: str(required=True)
+
+shirts-details:
+  url: str(required=True)
```

Comment:
Is this only required if the top-level string is present?

Reply:
Yes - that's why I changed the others too. If you specify shirt-details, it must have a URL and end, leaving out shirt-details entirely is fine.